app/src/main/java/org/mozilla/fenix/tabtray/TabTrayDialogFragment.kt

```diff
@@ -267,6 +275,20 @@ class TabTrayDialogFragment : AppCompatDialogFragment(), UserInteractionHandler
         }
     }
 
+    private fun showUndoSnackbarForTabs() {
```

Comment:
**Optional**: We could also refactor out https://github.com/mozilla-mobile/fenix/pull/16236/files#diff-7712cdb52aa96fe210fbd3cd4ca41b5007bb4d6043a4a857d2a8f77cbd38940aR310-R321 to call `showUndoSnackbarForTabs(snackbarMessage)`. There's quite a bit of duplication of the `tabsUseCases.undo` usage in a number of fragment. It would be nice to clean that up with a generic util function if possible.